Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM java:8
+
+RUN groupadd -r obelix && useradd -r -g obelix obelix
+
```

Comment:
What about forcing user ID of 1000 so that `docker-compose` can mount local volumes for local developments without any permission conflicts?

E.g. we usually do:

```
# Run container as user `flaskmenu` with UID `1000`, which should match
# current host user in most situations:
RUN adduser --uid 1000 --disabled-password --gecos '' flaskmenu && \
    chown -R flaskmenu:flaskmenu /code
```

see https://github.com/inveniosoftware/flask-menu/blob/master/Dockerfile#L30 and other inveniosoftware repositories.